packages/Button/src/index.tsx

```diff
@@ -9,7 +10,8 @@ interface IButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
   type?: TType;
   modifiers?: string[];
   utils: {
-    classNames: (className: string | object, classNames?: object) => string;
+    buildClassNames: (className: string | object, optionalClassNames?: object) => string;
+    useTheme: (componentName: string) => string
```

Comment:
Looks like the import was commented out and I can't see it on the withUtils function? Is this left over from testing?